src/client/app/components/meters/CreateMeterModalComponent.tsx

```diff
@@ -412,13 +411,19 @@ export default function CreateMeterModalComponent() {
 						{/* cumulativeReset input */}
 						<Col><FormGroup>
 							<Label for='cumulativeReset'>{translate('meter.cumulativeReset')}</Label>
-							<Input id='cumulativeReset' name='cumulativeReset' type='select'
-								value={meterDetails.cumulativeReset.toString()}
-								onChange={e => handleBooleanChange(e)}>
-								{Object.keys(TrueFalseType).map(key => {
-									return (<option value={key} key={key}>{translate(`TrueFalseType.${key}`)}</option>);
-								})}
-							</Input>
+							{meterDetails.cumulative === true ? (
+								<Input id='cumulativeReset' name='cumulativeReset' type='select'
+									value={meterDetails.cumulativeReset.toString()}
+									onChange={e => handleBooleanChange(e)}>
+									{Object.keys(TrueFalseType).map(key => {
+										return (<option value={key} key={key}>{translate(`TrueFalseType.${key}`)}</option>);
+									})}
+								</Input>
+							) : (
+								<Input id='cumulativeReset' name='cumulativeReset' type='select' disabled>
```

Comment:
If you set cumulative to yes, then cumulativeReset to yes and finally cumulative to no, then cumulativeReset shows unavailable but it stores to the DB as true (yes). This means that it is likely doing the reset at times. I think that if you change cumulative to no then it should modify the Redux state to make cumulativeReset be no so the DB gets the desired value. The same applies to edit.